pkg/controller/deployment/deployment_controller.go

```diff
@@ -415,6 +416,15 @@ func (dc *DeploymentController) syncDeployment(key string) error {
 		glog.V(4).Infof("Ignoring paused deployment %s/%s", d.Namespace, d.Name)
 		return nil
 	}
+	if d.Spec.RollbackTo != nil {
+		revision := d.Spec.RollbackTo.Revision
+		if err := dc.rollback(d, &revision); err != nil {
```

Comment:
Actually I decided to pass `&d` to rollback instead so that we can reconcile right after rolling-back.